### PR TITLE
Support placeholders (`$` or `?`) in `LIMIT` clause

### PR DESCRIPTION
**Pull request by step-baby**
#486

### CODE REVIEW

**Review by alamb — APPROVED** [review]
Looks good -- thank you @step-baby  - sorry for the delayed review

**Comment by alamb** [comment]
I took the liberty of fixing `cargo fmt` on this PR. Thanks again @step-baby

File: src/parser.rs [hunk 1]
```diff
@@ -4337,4 +4338,38 @@ mod tests {
             parser.prev_token();
         });
     }
+
+    #[test]
+    fn test_parse_limit() {
```
Comment (alamb) [inline src/parser.rs:4343]: I like these tests 👍

**Review by alamb — APPROVED** [review]

File: src/parser.rs (hunk 1 above)
Comment (step-baby) [reply src/parser.rs:4343]: thanks

**Pull request merged**